dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/mail/MailMessageCreator.java

```diff
@@ -69,22 +68,24 @@ public MailMessageCreator() {
     @Inject
     public MailMessageCreator(SystemConfig systemConfig, PermissionServiceBean permissionService,
                               DataverseDao dataverseDao, ConfirmEmailServiceBean confirmEmailService,
-                              GenericDao genericDao) {
+                              GenericDao genericDao, AuthenticatedUserRepository authenticatedUserRepository) {
         this.systemConfig = systemConfig;
         this.permissionService = permissionService;
         this.dataverseDao = dataverseDao;
         this.confirmEmailService = confirmEmailService;
         this.genericDao = genericDao;
+        this.authenticatedUserRepository = authenticatedUserRepository;
     }
 
     // -------------------- LOGIC --------------------
 
     /**
      * Creates footer for email message.
      */
-    public String createMailFooterMessage(Locale messageLocale, String rootDataverseName, InternetAddress systemAddress) {
-
-        return BundleUtil.getStringFromBundleWithLocale("notification.email.closing", messageLocale,
+    public String createMailFooterMessage(String notificationType, Locale messageLocale, String rootDataverseName, InternetAddress systemAddress) {
+        String closingLine = RETURNEDDS.equals(notificationType)
+                ? "notification.email.closing.other" : "notification.email.closing";
```

Comment:
I think that it is something wrong that you have to pass `notificationType` here. From what I understand some of the mails sent doesn't even have "notification nature". I would try maybe to leave the method as it was - maybe with renaming it to `createMailDefaultFooterMessage` and somewhere in higher level when we construct mail text for specific notification make a distinction

Reply:
I've moved that code one level higher, and also I've made a second version with `null` as notification type in order to hide that `null` in other places.